styles/globals.scss

```diff
@@ -26,7 +25,7 @@
 
 body {
   height: 100vh;
-  background-color: $SOFT_WHITE;
+  //background-color: $SOFT_WHITE;
```

Comment:
still need this?

Reply:
nope! I forgot I took this out locally but didn't push it up.